<launch>

<!-- launch qbo_random_move-->

<node name="qbo_random_move_explore" pkg="qbo_random_move" type="qbo_random_move_explore.py" output="screen">

<remap from="/cmd_vel" to="/qbo_random_move/obstacle_avoidance_twist"/>

</node>

<node name="qbo_random_move_avoid_obstacles" pkg="qbo_random_move" type="qbo_random_move_avoid_obstacles.py" output="screen">

<remap from="/output_twist" to="/cmd_vel"/>

</node>

</launch>

